.github/workflows/ubuntu-ci-x86_64-gnu.yaml

```diff
@@ -170,9 +170,9 @@ jobs:
           # Retain config from upstream so we don't have to rebuild:
           cp -r $PWD/envs/${ENVNAME}/{site,common} $PWD/envs/chaintest/.
```

Comment:
Since you changed $PWD/envs/${ENVNAME} for macOS CI, do you also want to do it for Ubuntu CI?